common/tools/eslint-plugin-azure-sdk/src/configs/azure-sdk-base.ts

```diff
@@ -88,7 +88,7 @@ export default {
     "@typescript-eslint/no-inferrable-types": "off",
     // We use empty extends and empty interface for shimming and renaming extensively
     "@typescript-eslint/no-empty-interface": "off",
-    "@typescript-eslint/no-namespace": "off",
+    "@typescript-eslint/no-namespace": "error",
```

Comment:
I wonder why it was turned off in the first place? but this is a question to the team :) @witemple-msft and @sadasant do you know?

Reply:
We had our own namespace rule which this PR removes in favor of this built-in one.

In the history of our eslint plugin, it was basically developed in 1:1 correspondence with the guidelines, and at that time the eslint/tslint/typescript-eslint story was not as set in stone as it is now where pretty much everyone is using eslint with the typescript plugin.